install_sct.bat

```diff
@@ -0,0 +1,66 @@
+:: Usage: install_sct.bat <version>
+:: e.g.
+::        install_sct.bat 5.5
+
+:: Turn off command echoing for nicer output
+@echo off
+
+:: Fetch version number (should be git branch, git tag, etc.)
+set git_ref=%1
+
+:: Go to user's home directory
+pushd %HOMEPATH%
+
+:: Download SCT and check out the branch requested by the user
+echo:

```

Comment:
I got the suggestion from [this SO answer](https://stackoverflow.com/a/132804), but given your docs link, I imagine that's a better way to go?

Reply:
Wow, after skimming [this forum discussion](https://www.dostips.com/forum/viewtopic.php?p=4554) very briefly, I don't know what to believe anymore. Maybe just keep `echo:`, since it's really not clear that any other suggestion is better, even the official one.